lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.ts

```diff
@@ -139,18 +139,8 @@ export class TaskListCloudComponent extends BaseTaskListCloudComponent {
         super(appConfigService, taskCloudService, userPreferences, PRESET_KEY);
     }
 
-    load(requestNode: TaskQueryCloudRequestModel) {
-        this.isLoading = true;
-        this.taskListCloudService.getTaskByRequest(requestNode).subscribe(
-            (tasks) => {
-                this.rows = tasks.list.entries;
-                this.success.emit(tasks);
-                this.isLoading = false;
-                this.pagination.next(tasks.list.pagination);
-            }, (error) => {
-                this.error.emit(error);
-                this.isLoading = false;
-            });
+    getTasks(requestNode: TaskQueryCloudRequestModel) {
+        return this.taskListCloudService.getTaskByRequest(requestNode);
```

Comment:
As this is the component and not a service i expect this one to subscribe to that method and not just simply return the Observable. Does this component needs this info? if yes how? is there an async pipe on html if so then we don't need a method but just get the observable from the service

Reply:
The point was : If a component subscribe to an observable is for the need of that information. If we just need to return the Observable of that fetching i would say at this point we don't need this anymore here? is it subscribed somewhere else? Let's have a call about this i think ti will be faster than writing 👯